ios/NativeSigner/Backend/ExportMultipleKeysService.swift

```diff
@@ -26,14 +26,39 @@ final class ExportMultipleKeysService {
         self.callbackQueue = callbackQueue
     }
 
-    func exportMultipleKeys(
-        items: [String]? = nil,
+    func exportMultipleKeySets(
+        seedNames: [String],
+        _ completion: @escaping (Result<AnimatedQRCodeViewModel, ServiceError>) -> Void
+    ) {
+        let selectedItems: [String: ExportedSet] = Dictionary(
+            uniqueKeysWithValues: seedNames
+                .map { ($0, ExportedSet.all) }
+        )
+        export(selectedItems: selectedItems, completion)
+    }
+
+    func exportRootWithDerivedKeys(
+        seedName: String,
```

Comment:
What if root key in those key details is not selected? Or do you need to add root key itself in keys list as well when it's selected?